problem_builder/instructor_tool.py

```diff
@@ -105,9 +129,59 @@ def student_view(self, context=None):
             _('Rating Question'): 'RatingBlock',
             _('Long Answer'): 'AnswerBlock',
         }
+        block_types = ('pb-mcq', 'pb-rating', 'pb-answer')
+        flat_block_tree = []
+
+        def build_tree(block, ancestors):
+            """
+            Build up a tree of information about the XBlocks descending from root_block
+            """
+            block_id = block.scope_ids.usage_id
+            # Block ID not in workbench runtime.
+            block_id = unicode(getattr(block_id, 'block_id', block_id))
+            block_name = getattr(block, "display_name", None)
+            block_type = block.runtime.id_reader.get_block_type(block.scope_ids.def_id)
```

Comment:
@itsjeyd crashes here with the traceback I've sent earlier. Works if replaced to `block.scope_ids.usage_id`